server/src/routes/roomsRoutes.ts

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import express from 'express';
+import { CommunicationIdentityClient } from '@azure/communication-identity';
+import { RoomsClient } from '@azure/communication-rooms';
+import { createRoom, joinRoom } from '../controllers/roomsController';
+
+export const roomsRouter = (identityClient: CommunicationIdentityClient, roomsClient: RoomsClient): express.Router => {
+  // Initialize router
+  const router = express.Router();
+
+  router.post('/', createRoom(identityClient, roomsClient));
+  router.post('/token', joinRoom(identityClient, roomsClient));
```

Comment:
Minor one: rename joinRoom to createToken or something like that

Reply:
I will update it to `getToken` instead of `joinRoom`